provisioning/service/test/_provisioningserviceclient_test.js

```diff
@@ -648,7 +648,30 @@ describe('ProvisioningServiceClient', function () {
       };
 
       var de = new ProvisioningServiceClient({ host: 'host', sharedAccessSignature: 'sas' }, fakeHttpHelper);
-      de.getAttestationMechanism(testEnrollmentId, testCallback);
+      de.getIndividualEnrollmentAttestationMechanism(testEnrollmentId, testCallback);
+    });
+  });
+
+  describe('#getEnrollmentGroupAttestationMechanism', function () {
```

Comment:
>describe [](start = 2, length = 8)

NIT, big nit: For both the individual and for the group I would have included the requirement item.